test/sdf/inertial_stats.sdf

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" ?>
+<!---
+Model consists of 4 cubes places symmetrically
+in the XY plane. This model is used to verity the
+"ign sdf --inertial-stats" tool .
+            +y
+            │
+          ┌─┼─┐
+       L3 │ │ │(0,5,0)
+          └─┼─┘
+            │
+  L2┌───┐   │     ┌───┐L1
+────┼┼┼┼┼───┼─────┼┼┼┼┼─── +x
+    └───┘   │     └───┘
+  (-5,0,0)  │     (5,0,0)
+          ┌─┼─┐
+          │ │ │(0,-5,0)
+          └─┼─┘
+          L4│
+-->
+<sdf version="1.6">
+
+    <model name="test_model">
+      <pose>0 0 0 0 0 0</pose>
+
+      <link name="link_1">
+        <pose>5 0 0 0 0 0</pose>
+        <inertial>
+          <mass>1.0</mass>
+          <inertia>
+            <ixx>1</ixx>
+            <iyy>1</iyy>
+            <izz>1</izz>
+          </inertia>
+        </inertial>
+        <collision name="collision_1">
+          <geometry>
+            <box></box>
```

Comment:
I recommend using physically meaningful sizes here. I typically use `mass = 12.0` for cubes of uniform density in order to simplify the math.

For example for cubes with side length of 1 and. mass = 12.0:

* ixx = ivy = izz = m/12 * (1 + 1) = 2

or with mass = 6.0:

* ixx = ivy = izz = 1

so please fill in the collision and visual sizes and make it consistent with an equivalent box of uniform density

Reply:
Done

Reply:
can you also fill in the box sizes in the collision and visual elements to match the equivalent size of the inertial?

Reply:
Done